### PR TITLE
DOP-3525: Options json support

### PR DESCRIPTION
**Pull request by mmeigs**
### Stories/Links:

DOP-3525

### Notes:

This PR enables all custom DOP options to be ingested from one local `options.json` file.

### CODE REVIEW

File: cli/index.ts [hunk 1]
```diff
@@ -439,24 +439,11 @@ function handleError(error: Error) {
...
-            `Encountered error:\n\n${versionPath}\n\nis not a file with a valid JSON object.`,
-          );
-          handleError(e);
-        }
-      }
       return options;
     case 'string':
+      console.log(__dirname);
+      console.log(existsSync(options));
+      console.log(options);
```
Comment (rayangler) [inline cli/index.ts:446]: Just curious if we're planning to leave this logging here

**Review by rayangler — APPROVED** [review]
It's cool that `versionData` seems to just work in terms of options parsing and normalization! LGTM!

File: cli/index.ts (hunk 1 above)
Comment (mmeigs) [reply cli/index.ts:446]: Oh jesus. My bad. Thank you!

File: cli/index.ts [hunk 2]
```diff
@@ -439,24 +439,11 @@ function handleError(error: Error) {
 function getObjectOrJSON(options) {
```
Comment (branberry) [inline cli/index.ts:439]: NIT: Not important, nor is it needed now, but it could be nice to add typing to this function at some point

**Review by branberry — APPROVED** [review]
Looks good to me! One tiny comment, but doesn't need to be addressed in this PR

File: cli/index.ts (hunk 2 above)
Comment (mmeigs) [reply cli/index.ts:439]: I'd be cool with adding that! For some reason, Redoc didn't feel it necessary. So, maybe we push this to the end of the project. Good find though!

**Pull request merged**